### PR TITLE
feat: add read only prop for input fields

### PR DESCRIPTION
**Pull request by StackTraceYo**
#6 

I would like to edit the card, but only if the CVV or Exp Changes. This allows for making the input fields not editable

### CODE REVIEW

**Comment by StackTraceYo** [comment]
@murilo-campaner please take a look. thanks

File: src/CreditCard.tsx [hunk 1]
```diff
@@ -256,6 +263,7 @@ const CreditCard = React.forwardRef<CreditCardType, CreditCardProps>(
         const numberMask = maskChars.join('');
 
         const cvvMask = ''.padStart(code.size, '9');
+        console.log(type);
```
Comment (murilo-campaner) [inline src/CreditCard.tsx:267]: remove the console.log

**Comment by murilo-campaner** [comment]
Also, please, fix lint =)

Anyway, its approved.

**Comment by alexnicita** [comment]
can we merge this please?

**Pull request merged**